core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -771,6 +771,24 @@ class WorkspaceApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  it should "return 400 if the bucket location requested is in an invalid format" in withTestDataApiServices { services =>
+    val newWorkspace = WorkspaceRequest(
+      namespace = testData.wsName.namespace,
+      name = "newWorkspace",
+      attributes = Map.empty,
+      authorizationDomain = None,
+      bucketLocation = Option("US")
```

Comment:
Just to confirm - when we clone a workspace that's currently in the `US` multi-region, we don't even specify the location, right?

Reply:
When a workspace is cloned, the location of the new workspace bucket is set to the same as the original (see https://github.com/broadinstitute/rawls/blob/d72a788693bf9892210f49871bdb7dfa16b148a0/core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala#L593).